tests/fixtures/smoke-tests/passing-test/tests/acceptance/acceptance-test.js

```diff
@@ -0,0 +1,18 @@
+import moduleForAcceptance from '../helpers/module-for-acceptance';
+import { test } from 'qunit';
+
+let beforeEachWorks;
+
+moduleForAcceptance('Module', {
+  beforeEach() {
+    beforeEachWorks = true;
```

Comment:
This would not catch the issue fixed in https://github.com/ember-cli/ember-cli/pull/4946.  Can you confirm that the arguments are correct?